Dockerfile

```diff
@@ -10,11 +10,11 @@ RUN go mod download
 COPY . .
 
 ARG VERSION
-ENV VERSION $VERSION
+ENV VERSION=$VERSION
```

Comment:
Why the switching from space-separated syntax to "=" here ?

Reply:
The space separated syntax is considered as legacy format
https://docs.docker.com/reference/build-checks/legacy-key-value-format/

Reply:
Indeed ! Thanks for the info :+1: